src/main/java/net/neoforged/moddevgradle/internal/PrepareRunOrTest.java

```diff
@@ -145,14 +147,37 @@ public void prepareRun() throws IOException {
         if (getRunTypeTemplatesSource().isEmpty()) {
             runConfig = resolveRunType(getSimulatedUserDevConfigForVanilla());
         } else {
-            var userDevConfig = UserDevConfig.from(getRunTypeTemplatesSource().getSingleFile());
+            var userDevConfig = loadUserDevConfig(getRunTypeTemplatesSource().getSingleFile());
             runConfig = resolveRunType(userDevConfig);
         }
 
         writeJvmArguments(runConfig);
         writeProgramArguments(runConfig);
     }
 
+    private UserDevConfig loadUserDevConfig(File userDevFile) {
```

Comment:
We can stop using the moddev-bundle config entirely at this stage.

Reply:
I meant that the userdev jar will contain the userdev config for the foreseeable future. Might as well resolve the userdev jar directly and load it from there.

HOWEVER I just noticed that for neodev we should still allow passing the userdev file directly without wrapping it in a dummy userdev jar. So nevermind.